openverse_api/docs/catalog.urls.rst

```diff
@@ -0,0 +1,29 @@
+catalog.urls package
```

Comment:
How much of this do we have to explicitly define? Is this something generated by sphinx that we just need to check in? Do you mind also adding some internal documentation that describes how to add a new module and what needs to be changed in the `docs/` folder when changes are made (if anything)?